provision/provision-extension-source.sh

```diff
@@ -12,6 +12,7 @@ if [[ false != "${NAME}" && false != "${REPO}" ]]; then
   # Clone or pull the extension repository
   if [[ ! -d "${DIR}/.git" ]]; then
     vvv_info " * Cloning the \"${NAME}\" extension, see \"${REPO}\""
+    git config --global --add safe.directory "${DIR}"
     git clone "${REPO}" --branch "${BRANCH}" "${DIR}" -q
```

Comment:
wouldn't `noroot git clone` do the job? Or `mkdir` then `chown` then `noroot git`? It's checking if DIR is owned by the current user, seeing it is not, and exiting

Reply:
My guess is that will work on new instances but on old will works as always? They can have different permissions.

Reply:
on virtualbox at least, the folder will have the ownership of the shared mount